channeld/channeld.c

```diff
@@ -140,16 +151,52 @@ struct peer {
 	/* If master told us to send wrong_funding */
 	struct bitcoin_outpoint *shutdown_wrong_funding;
 
-#if EXPERIMENTAL_FEATURES
 	/* Do we want quiescence? */
-	bool stfu;
+	bool stfu_request;
 	/* Which side is considered the initiator? */
 	enum side stfu_initiator;
 	/* Has stfu been sent by each side? */
 	bool stfu_sent[NUM_SIDES];
+	/* After STFU mode is enabled, wait for a signle message flag */
+	bool stfu_wait_single_msg;
 	/* Updates master asked, which we've deferred while quiescing */
 	struct msg_queue *update_queue;
-#endif
+	/* The opener side's share of post splice channel balance */
+	struct amount_sat splice_opener_funding;
+	/* The accepter side's share of post splice channel balance */
+	struct amount_sat splice_accepter_funding;
```

Comment:
change to msats now?

Reply:
This is switching to relative amounts in phase 2